src/Renderer/Text.php

```diff
@@ -138,7 +138,7 @@ function render(Frame $frame)
 
             switch ($text_deco) {
                 default:
-                    continue;
+                    break;
```

Comment:
Original intention may have been:

```suggestion
                    continue 2;
```

?